vss_layers/README.md

```diff
@@ -0,0 +1,243 @@
+protocol may offer data requests to be synchronous or asynchronous, or cached
+or on-demand.  While that is a feature and is likely part of the request sent
+via the protocol, it may be that some data items can support only one or the
+other, and that must then be defined for each data item.
+
+Whatever they may be, such details are deployment-specific and should be
+layered on top of the core model and catalog, not included in it.
+
+### Data classification
+
+An example of information that would be applicable only in some situations is to classify data into privacy sensitivity categories.
+Other classification reasons can be envisioned as well.
+
+Noteworthy for this example, and likely many others:
+
+1. What falls into each privacy category differs depending on country
+ jurisdiction, and perhaps sometimes on the usage situation.  The way data is
+ classified might differ between car brands.
+1. The same vehicle model may be sold in different markets and the same
+   product would then need different definitions on the individual level.
+
+The nature of local privacy laws (and likely many other examples) shows that
+such information cannot be defined as part of the common model or catalog --
+it simply differs too often between usage situations.  Sometimes it is
+required to view the system's data model with different layer configurations
+even within a single implementation.  Therefore, a layered approach is needed
+in the creation of a standard data model and data catalog.
+
+The fact that this may differ even on instances of vehicles of the same make
+shows that the ability to add/remove layers dynamically might be needed in the
+technology stack implementation.  Another consideration is for layers to be
+individually updated during software updates.
+
+### Access control
+
+A frequent augmentation of the data model information is definition of access
+permission rules.  These boil down to specific signals that a specific actor
+is allowed or not allowed to access.  Once again, this differs depending on
+the situation and should be defined as a separate layer.
+
+## General definition of VSS-layers
+
+The layer concept definition should impose very little restrictions.
+Conventions and particular tools may introduce their own restrictions
+for certain usage, at a later time.
+
+The layer concept itself is able to do all of the following:
+
+1. Modify existing metadata and ultimately redefine any signal (i.e. change
+   its metadata) compared to a previous definition for example the definition
+   in the standard catalog.
+1. Add new signals. It can be done in the private/ branch as recommended but the
+   mechanism allows adding them anywhere.
+1. Add new metadata, or relationships to new concepts for the purpose of
+   creating deployment-models, bindings to existing technology, classification
+   and many other things.
+1. Remove signals from the final data model (SYNTAX TO BE DEFINED)
```

Comment:
I know we discussed this, but I am not super-sure we need it.
After all,  an implementation can still give "404" or "40x not implemented" on signals from the "core" spec. I think it would be a more clean design, than letting layers "delete" anything (i.e. I would rather provide a deployment/stack specific layer that tags "not-implemented: true" to relevant leaves, and letting the implementation act accordingly)

Just an opinion though, I won't fight to the death for it

Reply:
Remove is really specific, something like _omit signals_ could work if we talk about management of nodes for implementation phase.